src/Tests/TestHelperTrait.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Nodes\Api\Tests;
+
+use App;
+use Nodes\Api\Auth\Auth;
+use PHPUnit_Framework_MockObject_MockObject;
+
+/**
+ * Api controllers tests helper.
+ *
+ * Use within tests of api controllers to call api routes as authenticated user, or as a guest.
+ *
+ * Here is an inline example:
+ * <code>
+ * <?php
+ * $this->being($user)->callApi('GET', '/api/users')->seeJson((new UserTransformer)->transform($user));
+ * ?>
+ * </code>
+ */
+trait TestHelperTrait
+{
+    /**
+     * @var PHPUnit_Framework_MockObject_MockObject|Auth
+     */
+    protected $authMock;
+
+    /**
+     * Act as provided user.
+     *
+     * This simulates user being authenticated with token.
+     *
+     * User does not need to be saved to database.
+     *
+     * @param $user
+     * @return $this
+     */
+    public function being($user)
+    {
+        $this->authMock = $this->getMockBuilder(Auth::class)->disableOriginalConstructor()->getMock();
+        $this->authMock->method('user')->willReturn($user);
+        App::instance('api.auth', $this->authMock);
+        return $this;
+    }
+
+    /**
+     * Visit the given URI with a JSON request.
+     *
+     * Api call will contain 'Accept' header required by nodes api package.
+     *
+     * To call api as authenticated user chain being() method first.
+     *
+     * @param  string $method
+     * @param  string $uri
+     * @param  array $data
+     * @param  array $headers
+     * @return $this
+     */
+    public function callApi($method, $uri, array $data = [], array $headers = [])
+    {
+        return $this->json($method, $uri, $data,
+            array_merge(['Accept' => 'application/vnd.nodes.v' . $this->getApiVersion() . '+json'], $headers));
```

Comment:
I mean, for me it's fine, I just want to make sure all cases has been thought through. Because you **can** change the vendor name in config files ([https://github.com/nodes-php/api/blob/master/config/settings.php](https://github.com/nodes-php/api/blob/master/config/settings.php)) and if they do, your trait would pretty much break, as for as I can see?

Sometimes a client takes over a project. Including the API codebase. And if they wanna change the `Accept` header to NOT contain `nodes` (there are some weird people out there), then they might change it in the config files.